src/tests/ftest/container/basic_snapshot.py

```diff
@@ -25,9 +25,9 @@
 import random
 
 from apricot import TestWithServers, skipForTicket
-from pydaos import DaosPool, DaosContainer, DaosSnapshot, DaosApiError
+from pydaos import DaosContainer, DaosSnapshot, DaosApiError
```

Comment:
(pylint-no-name-in-module) No name 'DaosContainer' in module 'pydaos'
(pylint-no-name-in-module) No name 'DaosSnapshot' in module 'pydaos'
(pylint-no-name-in-module) No name 'DaosApiError' in module 'pydaos'